packages/core/src/TableSection/TableSection.tsx

```diff
@@ -0,0 +1,56 @@
+import { forwardRef } from "react";
+
+import { HvBaseProps } from "../types/generic";
+import { ExtractNames } from "../utils/classes";
+import { useDefaultProps } from "../hooks/useDefaultProps";
+import { useUniqueId } from "../hooks/useUniqueId";
+import { staticClasses, useClasses } from "./TableSection.styles";
+import { HvSection, HvSectionProps } from "../Section";
+
+export { staticClasses as tableSectionClasses };
+
+export type HvTableSectionClasses = ExtractNames<typeof useClasses>;
+
+export interface HvTableSectionProps extends HvBaseProps<HTMLDivElement> {
+  /** Props to be passed to the underlying `HvSection` component. */
+  sectionProps?: HvSectionProps;
+  /** A Jss Object used to override or extend the styles applied to the empty state component. */
+  classes?: HvTableSectionClasses;
+}
+
+/**
+ * The `TableSection` component is a wrapper for the `Section` component that applies
+ * specific stylings for tables that follow the latest DS specifcations.
+ */
+export const HvTableSection = forwardRef<HTMLDivElement, HvTableSectionProps>(
+  (props, ref) => {
+    const {
+      id,
+      sectionProps,
+      classes: classesProp,
+      className,
+      children,
+    } = useDefaultProps("HvTableSection", props);
+
+    const { classes: sectionClasses, ...others } = sectionProps || {};
+    const { classes, cx } = useClasses(classesProp);
+    const elementId = useUniqueId(id, "hvTableSection");
+
+    return (
+      <HvSection
+        ref={ref}
+        id={elementId}
+        className={className}
+        classes={{
+          root: cx(classes.section, sectionClasses?.root),
+          header: cx(classes.sectionHeader, sectionClasses?.header),
+          actions: cx(classes.sectionActions, sectionClasses?.actions),
+          content: cx(classes.content, sectionClasses?.content),
```

Comment:
Having these 2 "layers" of  `useClasses` is confusion. We need a better way to address this 😞 

Maybe `HvTableSection` should be a `styled(HvTableSection)` - we're doing nothing besides overriding styles